Sources/Fuzzilli/FuzzIL/TypeCollectionStatus.swift

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Unit of code that can be executed, scored, and mutated.
```

Comment:
That is probably not the right comment? :D 
You can also move this enum into the Program Class if you want, or have a dedicated file for it, up to you

Reply:
oh copy paste without even reading it 🤦